packages/core/src/app/payment/storedInstrument/ManageInstrumentsModal.test.tsx

```diff
@@ -5,24 +5,23 @@ import {
     createCheckoutService,
     RequestError,
 } from '@bigcommerce/checkout-sdk';
-import { mount } from 'enzyme';
+import userEvent from '@testing-library/user-event';
 import React, { FunctionComponent } from 'react';
 
 import { createLocaleContext, LocaleContext, LocaleContextType } from '@bigcommerce/checkout/locale';
 import { CheckoutProvider } from '@bigcommerce/checkout/payment-integration-api';
+import {getYear} from '@bigcommerce/checkout/test-mocks';
```

Comment:
please add spaces

Reply:
Thanks for the feedback.
I have updated here and checked the whole PR using the search `import \{[a-z]`, no other occurence.